articles/libraries/machine_learning/concepts/Qccc.Introduction.md

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
In order for this article to appear in the documentation, you'll need to add the QML library folder to the `toc.yml` file at `articles/libraries/toc.yml`; the examples for numerics and chemistry are quite similar, so should be helpful here as well.

Reply:
yes, we need a toc.yml @alexeib2